app/helpers/application_helper/button/generic_object_definition_button_edit.rb

```diff
@@ -1,5 +1,5 @@
 class ApplicationHelper::Button::GenericObjectDefinitionButtonEdit < ApplicationHelper::Button::Basic
   def visible?
-    !(@display == 'generic_objects' || !@record.kind_of?(GenericObjectDefinition) || @actions_node)
+    !(@display == 'generic_objects' || !@record.kind_of?(GenericObjectDefinition))
```

Comment:
Is this class thill used after the changes?

Reply:
Addressed in the last commit - 11fc82a